demo/go/README.md

```diff
@@ -0,0 +1,76 @@
+# Picovoice Go Demos
+
+This Go module contains demos for processing real-time audio (i.e. microphone) and audio files using the Picovoice Platform.
+
+## Installation
+
+MicDemo uses [malgo](https://github.com/gen2brain/malgo) for cross-platform audio capture. It requires `cgo`, which on Windows may mean that you need to install a gcc compiler like [Mingw](http://mingw-w64.org/doku.php) to build it properly.
+
+## Usage
+
+NOTE: The working directory for the following go commands is:
+
+```console
+cd picovoice/demo/go
+```
+
+### File Demo
+
+It allows testing Picovoice on a corpus of audio files. The demo is mainly useful for quantitative performance benchmarking. It accepts 16kHz audio files. Picovoice processes a single-channel audio stream if a stereo file is
+provided it only processes the first (left) channel. The following processes a file looking for instances of the wake phrase defined in the file passed to the `-keyword_path` argument and then infers the follow-on spoken command
+using the context defined by the file passed to the `-context_path` argument:
+
+```console
+go run filedemo/picovoice_file_demo.go \
+-input_audio_path "path/to/input.wav" \
+-keyword_path "/path/to/keyword.ppn" \
+-context_path "/path/to/context.rhn"
```

Comment:
is there a -h or --help command for providing the args info?

Reply:
yes, this comes by default with the cli parser. I'll make a note in the docs